MOCC/src/configurations/NewPyTango/HealthMonitorDS.py

```diff
@@ -0,0 +1,39 @@
+from flask import Flask, jsonify
+from flasgger import Swagger
+
+
+app = Flask(__name__)
+app.config['SWAGGER'] = {
+    'title': 'Health Monitor API',
+    'uiversion': 2
+}
+Swagger(app)
```

Comment:
ask whatever you need to know.
Sorry, I misspoke, Flasgger is not just for the UI, but also for the documentation.
It's OK how you're binding the functions, but their yaml documentation is missing.
Without that Flasgger cannot create the UI.